site/src/utils/markdown-styles.js

```diff
@@ -25,39 +24,48 @@ export const code = styled.code<empty>(
   })
 )
 
-export const pre = styled.pre<empty>({
-  '& > code': {
-    padding: 0
+export const pre =
```

Comment:
multiple generic type params were adjusted in a wrong way in this file

Reply:
Hmm yeah, I'll revert this back and retry -- and any other page with `< empty` or `empty >`.